src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.tag;
+import java.util.ArrayList;
+import seedu.addressbook.data.person.ReadOnlyPerson;;
+
+/**
+ * Represents a tagging in the address book entry.
+ * 
+ *
+ */
+public class Tagging {
+	private ArrayList<String> taggings;
```

Comment:
Based on the class diagram given in the exercise, the list of `Tagging` objects should not be in this class.